agentconfig/agentconfig.go

```diff
@@ -670,3 +673,13 @@ func RestartFile() string {
 func UserAgent() string {
 	return "google-osconfig-agent/" + Version()
 }
+
+// DisableInventoryWrite returns if the DisableInventoryWrite setting is set.
+func DisableInventoryWrite() bool {
+	return disableInventoryWrite != "true" && disableInventoryWrite != "1"
```

Comment:
Should this be `disableInventoryWrite == "true" || disableInventoryWrite == "1"`? Or even `strings.EqualFold(disableInventoryWrite, "true") || disableInventoryWrite == "1"`?